### PR TITLE
Move to production

### PR DESCRIPTION
**Pull request by biwano**
More transformations to support Services in dash-app

Has to be merged in sync with https://github.com/KlimaDAO/dash-apps/pull/152

### CODE REVIEW

File: flows/fetch_S3_artifacts.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+""" Fetch artifacts from S3 and save them locally """
+import os
+from prefect.logging import get_run_logger
+import utils
+
+
+def fetch_S3_artifact(path):
```
Comment (0xAurelius) [inline flows/fetch_S3_artifacts.py:7]: this capitalization is weird, it should be `s3` to fit he Python snakecase standard

**Review by 0xAurelius — CHANGES_REQUESTED** [review]
Overall looks fine, identified one minor naming convention issue

But what's with the `_v2` suffixes? Are we pulling from a diff data source? Formatting differently?

**Comment by biwano** [comment]
> Overall looks fine, identified one minor naming convention issue
> 
> But what's with the `_v2` suffixes? Are we pulling from a diff data source? Formatting differently?

I put _v2 suffixes when I needed to keep compatibility for the legacy Dash App. So when there is a flow with two versions (one without _v2 and one with _v2) the one with the _v2 suffix is used by Dash Api. The one without _v2 is used by the legacy Dash app.

File: flows/fetch_S3_artifacts.py (hunk 1 above)
Comment (biwano) [reply flows/fetch_S3_artifacts.py]: Yeah, I corrected this.

**Comment by biwano** [comment]
> Overall looks fine, identified one minor naming convention issue
> 
> But what's with the `_v2` suffixes? Are we pulling from a diff data source? Formatting differently?

The _v2 suffixes are for compatibility purposes. When a file is used by the legacy dash app and dash-api. There is 
- one version without the suffix for the legacy dash_apps
- one version with the suffix for the dash-api.

**Review by 0xAurelius — APPROVED** [review]

**Pull request merged**